trino/client.py

```diff
@@ -855,7 +855,7 @@ def execute(
         call fetch() until finished is true.
         """
         if self.cancelled:
-            raise exceptions.TrinoUserError("Query has been cancelled", self.query_id)
+            raise exceptions.TrinoUserError({"message": "Query has been cancelled"}, self.query_id)
```

Comment:
Can you revert this line and put logic in the `TrinoQueryError` constructor to convert a str input into a dict if a str is passed into error?

Reply:
Done